authentik/lib/default.yml

```diff
@@ -106,7 +106,11 @@ cookie_domain: null
 disable_update_check: false
 disable_startup_analytics: false
 avatars: env://AUTHENTIK_AUTHENTIK__AVATARS?gravatar,initials
-geoip: "/geoip/GeoLite2-City.mmdb"
+events:
+  processors:
+    geoip: "env://AUTHENTIK_GEOIP?/geoip/GeoLite2-City.mmdb"
```

Comment:
do we want to deprecate AUTHENTIK_GEOIP, and add AUTHENTIK_GEOIP_CITY and AUTHENTIK_GEOIP_ASN instead?